webapp/src/App.js

```diff
@@ -63,35 +69,109 @@ function App() {
-            "stacks_contractDeploy",
-          ],
-          chains: [chain],
-          events: [],
+    if (chain.includes("stacks")) {
+      const { uri, approval } = await client.connect({
+        pairingTopic: undefined,
+        requiredNamespaces: {
+          stacks: {
+            methods: [
+              "stacks_signMessage",
+              "stacks_stxTransfer",
+              "stacks_contractCall",
+              "stacks_contractDeploy",
+            ],
+            chains: [chain],
+            events: [],
+          },
         },
-      },
-    });
+      });
 
-    if (uri) {
-      QRCodeModal.open(uri, () => {
-        console.log("QR Code Modal closed");
+      if (uri) {
+        QRCodeModal.open(uri, () => {
+          console.log("QR Code Modal closed");
+        });
+      }
+
+      const sessn = await approval();
+      setSession(sessn);
+      setChain(chain);
+      saveToLocalStorage("session", sessn);
+      saveToLocalStorage("chain", chain);
+      QRCodeModal.close();
+    } else {
+      const { uri, approval } = await client.connect({
+        pairingTopic: undefined,
+        requiredNamespaces: {
+          bip: {
+            methods: ["btc_transfer"],
+            chains: [chain],
+            events: [],
+          },
+        },
       });
+
+      if (uri) {
+        QRCodeModal.open(uri, () => {
+          console.log("QR Code Modal closed");
+        });
+      }
+
+      const sessn = await approval();
+      setSession(sessn);
+      setChain(chain);
+      saveToLocalStorage("session", sessn);
+      saveToLocalStorage("chain", chain);
+      QRCodeModal.close();
     }
+  };
+
+  const handleBtcTransfer = async () => {
+    try {
+      const address = session.namespaces.bip.accounts[0].split(":")[2];
+      const isMainnet = chain == chains[2];
+      const recipients = isMainnet
+        ? [
+            {
+              address: "3DP8pe2zJUcBezD35cLyZJGdbDwNYwBNtb",
+              amountSats: "20000",
+            },
+            {
+              address: "3Codr66EYyhkhWy1o2RLmrER7TaaHmtrZe",
+              amountSats: "30000",
+            },
+          ]
+        : [
+            {
+              address: "2NAm1LPPHQQ8AaLhXSYWrpApoCKcyjNJsjf",
+              amountSats: "20000",
+            },
+            {
+              address: "2Mx1h4VWiik8JNosa5nu4Gg96iPNQPJBWGa",
+              amountSats: "30000",
+            },
+          ];
+
+      const result = await client.request({
+        chainId: chain,
+        topic: session.topic,
+        request: {
+          method: "btc_transfer",
+          params: {
+            pubkey: address, //XXX: This one is required
+            recipients,
+          },
+        },
+      });
 
-    const sessn = await approval();
-    setSession(sessn);
-    setChain(chain);
-    saveToLocalStorage("session", sessn);
-    saveToLocalStorage("chain", chain);
-    QRCodeModal.close();
+      setResult({
+        method: "btc_transfer",
```

Comment:
let's follow the same naming convention as stacks methods:
`bitcoin_btcTransfer`